Resources/Maps/_NF/Shuttles/gazpacho.yml

```diff
@@ -0,0 +1,5718 @@
+      parent: 1
+  - uid: 14
+    components:
+    - type: Transform
+      pos: 8.5,1.5
+      parent: 1
+  - uid: 15
+    components:
+    - type: Transform
+      pos: 6.5,2.5
+      parent: 1
+  - uid: 16
+    components:
+    - type: Transform
+      pos: 4.5,2.5
+      parent: 1
+  - uid: 17
+    components:
+    - type: Transform
+      pos: 2.5,1.5
+      parent: 1
+- proto: GrilleDiagonal
+  entities:
+  - uid: 42
+    components:
+    - type: Transform
+      pos: -6.5,6.5
+      parent: 1
+  - uid: 43
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: -5.5,5.5
+      parent: 1
+  - uid: 44
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: -1.5,5.5
+      parent: 1
+  - uid: 45
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: -0.5,6.5
+      parent: 1
+- proto: GunSafeShuttleT2Spawner
+  entities:
+  - uid: 346
+    components:
+    - type: Transform
+      pos: -5.5,4.5
+      parent: 1
+- proto: Gyroscope
+  entities:
+  - uid: 121
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: -0.5,-3.5
+      parent: 1
+- proto: HeadBorgMining
+  entities:
+  - uid: 389
+    components:
+    - type: Transform
+      parent: 388
+    - type: Physics
+      canCollide: False
+    - type: InsideEntityStorage
+- proto: HolofanProjector
+  entities:
+  - uid: 459
+    components:
+    - type: Transform
+      pos: 3.7501082,-3.627493
+      parent: 1
+- proto: KitchenMicrowave
+  entities:
+  - uid: 614
+    components:
+    - type: Transform
+      pos: 1.5,1.5
+      parent: 1
+- proto: LampGold
+  entities:
+  - uid: 454
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: -3.4511342,5.4544287
+      parent: 1
+- proto: LeftArmBorgMining
+  entities:
+  - uid: 392
+    components:
+    - type: Transform
+      pos: -4.8058214,-4.5455713
+      parent: 1
+- proto: LeftLegBorgMining
+  entities:
+  - uid: 393
+    components:
+    - type: Transform
+      pos: -0.4710741,-6.514598
+      parent: 1
+- proto: LockerAtmosphericsFilled
+  entities:
+  - uid: 263
+    components:
+    - type: Transform
+      pos: -4.5,0.5
+      parent: 1
+- proto: LockerChiefEngineerFilled
+  entities:
+  - uid: 267
+    components:
+    - type: Transform
+      pos: -5.5,3.5
+      parent: 1
+- proto: LockerParamedicFilled
+  entities:
+  - uid: 261
+    components:
+    - type: Transform
+      pos: -4.5,1.5
+      parent: 1
+- proto: LockerPilotFilled
+  entities:
+  - uid: 266
+    components:
+    - type: Transform
+      pos: -1.5,4.5
+      parent: 1
+- proto: LockerSalvageSpecialistFilled
+  entities:
+  - uid: 265
+    components:
+    - type: Transform
+      pos: -4.5,-0.5
+      parent: 1
+- proto: LockerWallMedicalFilled
+  entities:
+  - uid: 512
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: 1.5,2.5
+      parent: 1
+- proto: MedicalBed
```

Comment:
Normal bed

Reply:
I feel like this can be negotiated, the medical bed is within the ships medical bay. Small as it may be, I have not yet seen a ship that doesnt have a medical bed in its medbay.

Reply:
Resolving for now, please bring this back up if it is still an issue.